packages/react/src/components/Tile/Tile.js

```diff
@@ -507,11 +507,23 @@ export class ExpandableTile extends Component {
         };
   }
 
+  resizeObserver = null;
+  resizeElement = React.createRef();
```

Comment:
Is `resizeElement` needed? I don't see it being used anywhere after this, so I'm a bit confused about it.

Reply:
yep! i'll take this out. artifact from some trouble shooting 👍🏾